api/src/main/java/io/druid/data/input/Row.java

```diff
@@ -76,7 +76,7 @@
   /**
    * Returns the metric column value for the given column name. This method is different from {@link #getRaw} in two
    * aspects:
-   *  1. If the column is absent in the row, numeric zero is returned, rather than null.
+   *  1. If the column is absent in the row, null is returned.
```

Comment:
It makes this method actually to behave the same as getRaw() on this regard, so this item should be removed?

Reply:
Also, please annotate this method and `getRaw()` with `@Nullable`

Reply:
done